meta_view.php

```diff
@@ -208,7 +208,11 @@
    # above the image. Not easy with template blocks.
    $cols=5;
    $i = 0;
-   $count=count($names);
+   if (is_null($names)) {
+      $count=0;
+   } else {
+      $count=count($names);
+   }
```

Comment:
fix: PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /usr/share/ganglia/meta_view.php:211